src/components/AddInput/AddInput.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import './AddInput.css';
+import { v4 } from 'uuid';
+
+function AddInput({
+  // eslint-disable-next-line react/prop-types
```

Comment:
Please don't remove linters,

- [ ] You can add prop types using ```import PropTypes from 'prop-types;```
And then `AddInput.PropTypes = {
  //variables
}
`